core/src/main/java/org/mule/runtime/core/api/policy/PolicyChain.java

```diff
@@ -35,7 +35,7 @@
 import org.mule.runtime.core.api.processor.strategy.ProcessingStrategy;
 import org.mule.runtime.core.internal.context.notification.DefaultFlowCallStack;
 import org.mule.runtime.core.internal.exception.MessagingException;
-import org.mule.runtime.core.internal.policy.PolicyNotificationHelper;
+import org.mule.runtime.core.api.policy.PolicyNotificationHelper;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary import: same package classes are always implicitly imported. ([squid:UselessImportCheck](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7760&issues=AWnGD4_aotIXnmsUaS7h&open=AWnGD4_aotIXnmsUaS7h)

[ISSUE_KEY]: AWnGD4_aotIXnmsUaS7h